gobblin-service/src/main/java/org/apache/gobblin/service/modules/spec/JobExecutionPlanListSerializer.java

```diff
@@ -82,6 +82,8 @@ public JsonElement serialize(List<JobExecutionPlan> src, Type typeOfSrc, JsonSer
 
       String executionStatus = jobExecutionPlan.getExecutionStatus().name();
       jobExecutionPlanJson.addProperty(SerializationConstants.EXECUTION_STATUS_KEY, executionStatus);
+      jobExecutionPlanJson.addProperty(SerializationConstants.CURRENT_GENERATION_KEY, jobExecutionPlan.getCurrentGeneration());
```

Comment:
are these two fields always present in jobExecutionPlan or do they need a default value?